DemoQA/cypress/pages/regisform/regisForm.js

```diff
@@ -0,0 +1,96 @@
+
+
+export const regisForm ={
+
+    TXT_FIRSTNAME: "#firstName",
+    TXT_LASTNAME: "#lastName",
+    TXT_EMAIL: "#userEmail",
+    RDO_GENDER: "#gender-radio-2",
```

Comment:
You should make them more flexible so you can choose any options you want, also for hobbies below